scripts/main.js

```diff
@@ -70,5 +70,62 @@ const departments = {
         ],
     }
 }
+//opdracht 1
+console.log("De afdeling Sales heeft " + departments.sales.numberOfEmployees + " medewerkers");
+console.log("Marketing is een leuke afdeling om te werken" + departments.marketing.description);
+console.log("De afdeling Customer Service heeft " + departments["customer-service"].numberOfEmployees + " medewerkers");
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. " + departments.sales.jobs[1].description);
+console.log(departments);
+
+//opdracht 2a
+const userInput = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]?');
+console.log(userInput);
+
+//opdracht 2b
+switch (userInput) {
+    case "marketing":
+        console.log("Je koos " + userInput + ". " + departments.marketing.description);
+        break
+    case "sales":
+        console.log("Je koos " + userInput + ". " + departments.sales.description);
+        break
+    case "customer-service":
+        console.log("Je koos " + userInput+ ". " + departments["customer-service"].description);
+        break
+    default:
+        console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");
```

Comment:
"Probeer het opnieuw door de pagina te verversen" Nette toevoeging!

Reply:
If statement is inderdaad minder code. Ik weet niet wat best practice is, wellicht kan één van ons het vragen in de volgende les. Het werkt beide iig.